crates/consensus/src/transaction/eip4844/builder.rs

```diff
@@ -3,6 +3,14 @@ use alloy_eips::eip4844::Blob;
 #[cfg(feature = "kzg")]
 use c_kzg::{Blob, KzgCommitment, KzgProof};
 
+#[cfg(not(feature = "std"))]
+use alloc::vec::Vec;
+#[cfg(not(feature = "std"))]
+use core::cmp;
```

Comment:
no need to feature gate core ever as it is part of both the `std` prelude and the `core` prelude.

Reply:
Yes please just use `core` and `alloc` instead of feature gating imports

Reply:
alloc needs to be feature gated due to new redundant import lints